server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -217,10 +223,16 @@ public String toString() {
 
     public class PodSpec {
         public String name = "";
+        public String moduleName = "";
         public String version = "";
         public String originalJSON = "";
         public String iosversion = "";
         public String osxversion = "";
+        public String iosModuleMap = null;
+        public String osxModuleMap = null;
+        public Boolean isFramework = false;
+        public Set<String> swiftSourceFilePaths = new LinkedHashSet<>();
```

Comment:
We need the swift source files in a separate set since we want to build those first, and then build other source files.